docs/source/components/pagination/content.md

```diff
@@ -42,3 +42,6 @@ If `true`, renders pagination content inside an infinite scroll component
 
 ### `infiniteScrollProps?: InfiniteScrollProps`
 Only used when `infiniteScroll` is true. See [react-infinite-scroll-component](https://github.com/ankeetmaini/react-infinite-scroll-component#props)
+
+### `render?: ({ ...UsePaginationReturnType, ...PaginationContentProps }) => ReactNode`
```

Comment:
If there's a better way to define this, please let me know. We could explicitly put all the properties of the `usePagination` return type and all of the prop types for `PaginationContent` but I was concerned about updates being made to one and not the other.

Reply:
I think I get my words backwards when I say `render` prop. I was more so thinking instead of a render prop we directly let you make the children have the props like: https://github.com/Availity/availity-react/blob/master/packages/spaces/src/Spaces.js#L132-L134

For `PaginationContent` I would just worry about the props needed to render that piece. If the user wants to do anything outside of that they can just build their own custom component at that point which can do everything they are looking for.

Reply:
56e566a375c61f399fde2bab398e1f9a543de031